.ci/releasePipeline.groovy

```diff
@@ -112,7 +130,22 @@ pipeline {
                                                 pushToGithub(rleaseName: release + ".sha256", source: "${release}/${release}.sha256", releaseId: releaseId)
                                                 break
                                             case "container":
-                                                createContainerAndPushToStorage(version: version, registrySecretsPath: 'keptn-jenkins/monaco/registry-deploy')
+                                                // docker hub, only full releases and rc-releases
+                                                if (!isDevRelease) {
+                                                    createContainerAndPushToStorage(version: version, registrySecretsPath: 'keptn-jenkins/monaco/dockerhub-deploy', registry: 'Docker')
+
+                                                    // push latest to full releases and rc-releases
+                                                    createContainerAndPushToStorage(version: 'latest', registrySecretsPath: 'keptn-jenkins/monaco/dockerhub-deploy', registry: 'Docker')
+                                                } else {
+                                                    echo "Skipping release of ${version} to docker"
```

Comment:
my improvement point was actually more about "docker hub" rather than "docker", also logging the version was just a bonus :D